### PR TITLE
Feat/users

### PR DESCRIPTION
**Pull request by Ajay872**

### CODE REVIEW

File: application/frontend/staff/staff.html [hunk 1]
```diff
@@ -253,9 +286,8 @@ <h2 class="mt-4 text-xl font-semibold text-gray-200" id="userId"></h2>
         <p
           class="mt-4 font-[comfortaa] text-[--light] text-lg"
         >
-          As a Staff member, you can register Patients in the Portal and Schedule their Appointments with Doctors.
+          As a Admin, you can Manage Doctor Appointments, Manage Users, See All Patient Data and Generate Reports as well...
```
Comment (yash797) [inline application/frontend/staff/staff.html:289]: admin why??

File: application/frontend/staff/staff.html [hunk 2]
```diff
@@ -188,45 +188,78 @@ <h2 class="mt-4 text-xl font-semibold text-gray-200" id="userId"></h2>
 
     <!-- //bottombar -->
     <div
-      class="fixed md:invisible bottom-0 left-0 z-50 w-full h-16 bg-white border-t border-gray-200 dark:bg-gray-700 dark:border-gray-600"
+      class="fixed md:invisible bottom-0 left-0 z-50 w-full h-16 bg-dark hover:bg-dark   border-gray-200 dark:bg-gray-700 dark:border-gray-600"
```
Comment (yash797) [inline application/frontend/staff/staff.html:191]: remove this dark theme style

Comment (Ajay872) [reply application/frontend/staff/staff.html:191]: done

File: application/frontend/staff/staff.html (hunk 1 above)
Comment (Ajay872) [reply application/frontend/staff/staff.html]: done

**Review by yash797 — APPROVED** [review]

**Pull request merged**